src/Graphics/OpenGLContext/opengl_GLInfo.cpp

```diff
@@ -61,7 +60,7 @@ void GLInfo::init() {
 	if (isGLESX && bufferStorage)
 		g_glBufferStorage = (PFNGLBUFFERSTORAGEPROC) eglGetProcAddress("glBufferStorageEXT");
 #endif
-	texStorage = (isGLESX && (numericVersion >= 30)) || (!isGLESX && numericVersion >= 42) ||
+	texStorage = (isGLESX && (numericVersion >= 30) && config.video.multisampling != 0) || (!isGLESX && numericVersion >= 42) ||
```

Comment:
I think the reason I did this was because I noticed some performance issues with texStorage. So I only enabled it in the case where it was necessary. I think I could revert that change.